### DISCUSSION TITLE
NPC indicator - tag by level

### DISCUSSION THREAD

**Discussion by deejcunningham** [runelite/runelite#15429] [Q&A] [answered]
Two kinds of Dark wizard spawn at Delrith's circle south of Varrock: level-7 and level-20. The level-20 Dark wizards have slightly better drop rates for clue scrolls, so are preferable to kill. However, using NPC indicator to highlight "Dark wizard" highlights both levels.

It would be convenient if NPC indicator could differentiate NPCs by their level instead of only by name. I.e. "Dark wizard" would highlight both levels, but "Dark wizard (level-20)" would only highlight the higher level Dark wizards.

**Comment by Nightfirecat** [comment, marked as answer]
You can use a hub plugin such as [Better NPC Highlight](https://runelite.net/plugin-hub/show/better-npc-highlight) to highlight NPCs by ID. We're not presently interested in adding further highlight specificity for the core plugin.
